UPGRADE-5.4.md

```diff
@@ -69,6 +69,13 @@ Messenger
  * Deprecate not setting the `delete_after_ack` config option (or DSN parameter) using the Redis transport,
    its default value will change to `true` in 6.0
 
+
+Notifier
+--------
+
+* The following data providers for `TransportTestCase` are now static: `toStringProvider()`, `supportedMessagesProvider()` and `unsupportedMessagesProvider()`
```

Comment:
This is already done in https://github.com/symfony/symfony/pull/49368

Reply:
It was in case the main PR is not ready for the next patch version of 5.4. But I guess we'll be able to tackle everything before next release so I'll remove it! 🙂